packages/libiconv.rb

```diff
@@ -3,32 +3,27 @@
 class Libiconv < Package
   description 'GNU charset conversion library for libc which does not implement it.'
   homepage 'https://www.gnu.org/software/libiconv/'
-  version '1.15'
+  version '1.15-1'
   source_url 'https://ftpmirror.gnu.org/libiconv/libiconv-1.15.tar.gz'
   source_sha256 'ccf536620a45458d26ba83887a983b96827001e92a13847b45e4925cc8913178'
 
   binary_url ({
-    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/libiconv-1.15-chromeos-armv7l.tar.xz',
-     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/libiconv-1.15-chromeos-armv7l.tar.xz',
-       i686: 'https://dl.bintray.com/chromebrew/chromebrew/libiconv-1.15-chromeos-i686.tar.xz',
-     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/libiconv-1.15-chromeos-x86_64.tar.xz',
   })
   binary_sha256 ({
-    aarch64: '047163a67f3ea9e143b465087d5ea4a3d17b963c8ad3579bc27a21934d4a17ab',
-     armv7l: '047163a67f3ea9e143b465087d5ea4a3d17b963c8ad3579bc27a21934d4a17ab',
-       i686: '28de005b59407d5343775296814e10272c53aedd860e0208f396cc873045c095',
-     x86_64: '8689ee2b4acc0440df6eec9eb0fb19aa8393b68bd64517cbca65d98041a2c7cc',
   })
   
-  depends_on 'glibc'
-  depends_on 'manpages'
-
   def self.build
-    system "./configure --prefix=#{CREW_PREFIX} --libdir=#{CREW_LIB_PREFIX}"
-    system "make"
+    system './configure',
+           '--enable-shared',
+           '--enable-static',
```

Comment:
Why do we need static libraries?

Reply:
Statically linking the core packages so as to reduce dependencies.

Reply:
This update is still sitting here since I thought you were going to update ruby.

Reply:
Yeah, I know. Ruby's having some issues doing this right now, look:
```
~snip~
in `require': incompatible library version
~snip~
```

Reply:
And this is the exactly the reason I close these kinds of PRs.  I waste a lot of my time testing PRs that break other packages.

Reply:
This doesn't break anything. It's just Ruby that's being a pain during build.

Reply:
You just contradicted yourself.  This breaks ruby.

Reply:
No, Ruby breaks Ruby. This is fine.